windows/QMK Toolbox/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("QMK")]
 [assembly: AssemblyProduct("QMK Toolbox")]
-[assembly: AssemblyCopyright("Copyright ©  2017 Jack Humbert")]
+[assembly: AssemblyCopyright("Copyright ©  2019 Jack Humbert")]
```

Comment:
Software copyrights usually start at the initial year of the application and end at current year, because copyright goes into effect when the work is created, regardless of when it's released.

```suggestion
[assembly: AssemblyCopyright("Copyright ©  2017-2019 Jack Humbert")]
```

Reply:
Okay

Reply:
My point here was the copyright dates should be `2017-2019`, rather than just one or the other.